lualib/lua_fuzzy.lua

```diff
@@ -246,7 +246,7 @@ local function mime_types_check(task, part, rule)
   local id = part:get_id()
   lua_util.debugm(N, task, 'check binary part %s: %s', id, ct)
 
-  -- For bad mime mime parts we implicitly enable fuzzy check
+  -- For bad mime parts we implicitly enable fuzzy check
```

Comment:
Duplicate word detection is a thing I'm experimenting with. I think these changes are correct...

Reply:
They are!